opencga-app/app/scripts/azure/arm/mongodb/postdeploy.sh

```diff
@@ -102,18 +137,22 @@ createReplicaSet() {
 
 #install flow
 installDeps
+scanForNewDisks
 generateCertificate
 configureMongoDB
 
-# only for primary mongodb instance
-if [ $VM_INDEX -eq 0 ]
+# only configure replicasets when clusterSize > 1
+if [ $CLUSTER_SIZE -gt 1 ]
 then
-    generateMongoKeyFile
+    if [ $VM_INDEX -eq 0 ]
+    then
+        generateMongoKeyFile
 
-    #waiting for other mongodb instances to be successfully configured
-    sleep 120
+        #waiting for other mongodb instances to be successfully configured
+        sleep 120
```

Comment:
Is this number constant no matter how many instances in the replica set?

Reply:
since all VMs are deployed asynchronously it shouldn't matter on how many VMs it's waiting for